src/OmniSharp.Http/Startup.cs

```diff
@@ -45,8 +52,15 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                                 !category.Equals(projectEventForwarder, StringComparison.OrdinalIgnoreCase)));
                 });
 
+            var options = serviceProvider.GetRequiredService<IOptionsMonitor<OmniSharpOptions>>();
+            var plugins = new PluginAssemblies(_commandLinePlugins.Concat(options.CurrentValue.Plugins.GetNormalizedLocationPaths(_environment)));
+
+            var loggerFactory = serviceProvider.GetRequiredService<ILoggerFactory>();
+            var logger = loggerFactory.CreateLogger<Startup>();
+            var assemblyLoader = serviceProvider.GetRequiredService<IAssemblyLoader>();
             _compositionHost = new CompositionHostBuilder(serviceProvider)
                 .WithOmniSharpAssemblies()
+                .WithAssemblies(assemblyLoader.LoadByAssemblyNameOrPath(logger, plugins.AssemblyNames).ToArray())
```

Comment:
`
var assemblies = assemblyLoader.LoadByAssemblyNameOrPath(logger, plugins.AssemblyNames).ToArray();
`
Then:
`
    .WithAssemblies(assemblies);
`

Reply:
Mimicked existing order for the statements as in src\OmniSharp.LanguageServerProtocol\LanguageServerHost.cs and src\OmniSharp.Stdio.Driver\Program.cs